Package.swift

```diff
@@ -3,14 +3,18 @@ import PackageDescription
 
 let package = Package(
     name: "StripeKit",
+    platforms: [
+        .macOS(.v10_15)
+    ],
     products: [
         .library(name: "StripeKit", targets: ["StripeKit"])
     ],
     dependencies: [
         .package(url: "https://github.com/swift-server/async-http-client.git", .exact("1.1.0")),
```

Comment:
I think it’s safe to trust the NIO guys and use `.from`. I know I originally added it but would be appreciated if you could make that small change :)